src/components/IndexTabs/IndexTab.module.scss

```diff
@@ -25,18 +25,29 @@
   color: var(--rhino);
   justify-content: space-between;
 
-  @include text('h4', 'm');
+  @include text("h4", "m");
 
-  @include responsive('phone', 'phone-xs') {
+  @include responsive("phone", "phone-xs", "tablet") {
```

Comment:
To decrease the complexity of source/generated css code we can use the classes provided by the [`webkit/responsive`](https://github.com/santiment/san-webkit/blob/master/src/responsive.ts#L22) script. 
For example, in this case it could be written as following: 
```scss 
.phone-xs &, .phone &, .tablet & { ... }
```
To make it possible, we have to import the [`webkit/responsive`](https://github.com/santiment/san-webkit/blob/master/src/responsive.ts#L22) script in the `src/index.js`